README.md

```diff
@@ -5,14 +5,37 @@ Just a simple pecan app
 
 1. clone the repo
 
-2. install pecan in virtual env: https://pecan.readthedocs.io/en/latest/installation.html
+2. cd flocx-market and run cmd:
+```
+pip install -r requirements.txt
+```
+3. migrate the database:
+```
+cd flocx_market/db/sqlalchemy
+```
+```
+alembic upgrade head
+```
+
+# List available offers
+1. First create a offer record in the marketplace database, offer table (from MySQL cmd line)
+```
+INSERT INTO offer VALUES (1,'b711b1ca-a77e-4392-a9b5-dc84c4f469ac','b9752cc0-9bed-4f1c-8917-12ade7a6fdbe','Alice1992','2016-07-16T19:20:30','available','Dell380','2016-07-16T19:20:30','2016-08-16T19:20:30',164,' {
+    "new attribute XYZ": "This is just a sample list of free-form attributes used for describing a server.",
+    "cpu_type": "Intel Xeon",
+    "cores": 16,
+    "ram_gb": 512,
+    "storage_type": "samsung SSD",
+    "storage_size_gb": 204
+  }',11);
+```
+2. Run the pecan app:
+```
+PYTHONPATH=. python flocx_market/cmd/api.py
```

Comment:
This project should install a script named something like `flocx-market-api` that would work without having to set `PYTHONPATH` or directly run a `.py` file like this.

Reply:
I looked at esi-leap code. In readme file, Mainn says 'run the service by:  sudo esi-leap-api'. I don't know if this is a shell script or something. I cannot find it in his repo.

Reply:
It is a Python script that is created as part of the install process because of the entries in the `entrypoints` section of `setup.cfg`: https://github.com/CCI-MOC/esi-leap/blob/master/setup.cfg#L15-L18

Each of the lines under `console_scripts` defines a command line script that will be installed. You can read a little bit more about entry points [here](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point).